packages/webapp/src/stories/Filters/FilterMultiSelectV2.stories.jsx

```diff
@@ -0,0 +1,69 @@
+/*
+ *  Copyright (c) 2024 LiteFarm.org
+ *  This file is part of LiteFarm.
+ *
+ *  LiteFarm is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  LiteFarm is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU General Public License for more details, see <https://www.gnu.org/licenses/>.
+ */
+
+import { componentDecorators } from '../Pages/config/Decorators';
+import { FilterMultiSelect } from '../../components/Filter/FilterMultiSelectV2';
+
+const props = {
+  subject: 'Status',
+  options: [
+    {
+      value: 'active',
+      default: false,
+      label: 'Active',
+    },
+    {
+      value: 'planned',
+      default: false,
+      label: 'Planned',
+    },
+    {
+      value: 'complete',
+      default: false,
+      label: 'Complete',
+    },
+    {
+      value: 'abandoned',
+      default: false,
+      label: 'Abandoned',
+    },
+    {
+      value: 'needs_plan',
+      default: false,
+      label: 'Needs plan',
+    },
+  ],
+  onChange: () => {
+    console.log('onChange fired!');
+  },
+  shouldReset: 0,
+};
+
+export default {
+  title: 'Components/Filters/FilterMultiSelect',
+  component: FilterMultiSelect,
```

Comment:
`component: FilterMultiSelectV2`

Reply:
@SayakaOno do you think we should create a story for the original FilterMultiSelect as well since it wasn't deleted?

Reply:
I don't think it's a must, but now that you mentioned, I think it would be nice to have one!